charts/bitwardenrs/1.1.1/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- DirtyCajunRice
+reviewers:
+- DirtyCajunRice
```

Comment:
We do not use OWNERS file at the moment (it's on the radar), so this file is prefered to be removed